### PR TITLE
Split Chatbox in 2 and continued chatbot

### PR DESCRIPTION
**Pull request by ArielGGutierrez**
We had been discussing that we should split the chatbox page into one for business, and one for user.  I've done that here and continued working on the chatbot.  I didn't delete the previous one-page implementation for the sake of security, and also haven't touched the business page.

image

### CODE REVIEW

File: portal/src/ChatBox/Business/ChatBot.js [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+    }, [])
+
+    const buttonQuestions = [
+        "Hello!",
+        "How are you?",
+        "Are you open?",
+        "Are you closed?",
+        "Can I talk to a real person?"
+    ];
+
```
Comment (NikitaSalikov) [inline portal/src/ChatBox/Business/ChatBot.js:19]: Should Business have the chatbot though?

**Review by NikitaSalikov — APPROVED** [review]
Looks good to me. Left a comment regarding the need for chat bot on Business side

**Comment by jmyanASU** [comment]
Approved

**Review by jmyanASU — APPROVED** [review]
Love it!

**Pull request merged**